packages/abi-typegen/src/utils/getFunctionInputs.test.ts

```diff
@@ -0,0 +1,134 @@
+import type { IType } from '../types/interfaces/IType';
+import type { JsonAbiArgument, JsonAbiType } from '../types/interfaces/JsonAbi';
+
+import { getFunctionInputs } from './getFunctionInputs';
+import { makeType } from './makeType';
+
+const nonEmptyType: JsonAbiType = {
+  type: 'u8',
+  typeId: 1,
+  components: null,
+  typeParameters: null,
+};
+
+const voidAbiType: JsonAbiType = {
+  type: '()',
+  typeId: 2,
+  components: null,
+  typeParameters: null,
+};
+
+const optionAbiType: JsonAbiType = {
+  type: 'enum Option',
+  typeId: 3,
+  components: null,
+  typeParameters: null,
+};
+
+const debugOptionAbiType: JsonAbiType = {
+  type: 'enum std::option::Option',
+  typeId: 4,
+  components: null,
+  typeParameters: null,
+};
+
+const EMPTY_ABI_TYPES: [string, JsonAbiType][] = [
+  ['void', voidAbiType],
+  ['option (release)', optionAbiType],
+  ['option (debug)', debugOptionAbiType],
+];
+
+const types: Array<IType> = [nonEmptyType, voidAbiType, optionAbiType, debugOptionAbiType].map(
+  (rawAbiType) => makeType({ rawAbiType })
+);
+
+/**
+ * @group node
```

Comment:
Should be browser compatible 

```suggestion
 * @group node
 * @group browser
```

Reply:
2f5b4bb7399a268b9d736ed73bddda15403d0fc2